packages/collector/Collector.php

```diff
@@ -0,0 +1,229 @@
+<?php
+/**
+ * @package Collector
+ * @version 0.0.0
+ */
+/*
+Plugin Name: Collector
+Plugin URI: https://github.com/seanmorris/collector
+Description: Packages your WordPress install and sends it to Playground.
+Author: Sean Morris
+Version: 0.0.0
+Author URI: https://github.com/seanmorris/
+*/
+
+const COLLECTOR_DOWNLOAD_PATH   = '/wp-admin/?page=collector_download_package';
+const COLLECTOR_FINAL_ZIP       = '/tmp/collector-package.zip';
+
+define('COLLECTOR_PLAYGROUND_PACKAGE', ($_SERVER['SERVER_NAME'] === 'localhost')
+	? 'http://localhost:8081/index.js'
+	: 'https://unpkg.com/@wp-playground/client/index.js'
```

Comment:
I feel uneasy about shipping an unpkg url here. I'd rather do one of these:

* Ship that file on playground.wordpress.net and use that URL here. I'm not sure if that will pass the plugin review, but it should – there are many plugins relying on external scripts.
* Ship `client/index.js` with this plugin – although that will add some maintenance burden to keep releasing new plugin versions whenever the client library gets updated

Reply:
@adamziel How should we proceed here?

Reply:
https://github.com/WordPress/wordpress-playground/issues/778

Reply:
@adamziel THANK YOU!!!!!!!!